test/Crate/Test/DBAL/Platforms/CratePlatformTest.php

```diff
@@ -318,7 +438,7 @@ public function testGenerateArraySQLDeclaration()
 
     public function testPlatformSupport() {
         $this->assertFalse($this->_platform->supportsSequences());
-        $this->assertTrue($this->_platform->supportsSchemas());
+        $this->assertFalse($this->_platform->supportsSchemas());
```

Comment:
ok so we'll have to do this, then disabling makes sense ;)
thx for pointing at this.